classes/class-wc-rest-connect-shipping-rates-controller.php

```diff
@@ -18,13 +18,32 @@ class WC_REST_Connect_Shipping_Rates_Controller extends WC_REST_Connect_Base_Con
 	 */
 	public function post( $request ) {
 		$payload = $request->get_json_params();
+		$payload[ 'payment_method_id' ] = $this->settings_store->get_selected_payment_method_id();
 		$order_id = $request[ 'order_id' ];
 
 		// This is the earliest point in the printing label flow where we are sure that
 		// the merchant wants to ship from this exact address (normalized or otherwise)
 		$this->settings_store->update_origin_address( $payload[ 'origin' ] );
 		$this->settings_store->update_destination_address( $order_id, $payload[ 'destination' ] );
 
+		// Update the customs information on all this order's products
+		$updated_product_ids = array();
+		foreach ( $payload[ 'packages' ] as $package_id => $package ) {
+			if ( ! isset( $package[ 'contents_type' ] ) ) {
```

Comment:
Relying on `contents_type` seems a bit arbitrary, similar to using a magic number. Would be cool to extract this out to a constant or function like `packageNeedsCustoms()`. That way, the comment in the line below (`// No customs information in this order, bail) would not be needed, since the function will be self-documenting.

Reply:
Done in https://github.com/Automattic/woocommerce-services/pull/1382/commits/73be1a127ea1375ffb868e10aed3139db799b926